climpred/prediction.py

```diff
@@ -5,18 +5,27 @@
 import numpy as np
 import xarray as xr
 
-from xskillscore import mae as _mae
-from xskillscore import mse as _mse
-from xskillscore import pearson_r as _pearson_r
-from xskillscore import pearson_r_p_value
-from xskillscore import rmse as _rmse
-
-from .comparisons import (_e2c, _e2r, _m2c, _m2e, _m2m, _m2r,
+from .comparisons import (_drop_members, _e2c, _e2r, _m2c, _m2e, _m2m, _m2r,
                           get_comparison_function)
-from .metrics import _nmae, _nmse, _nrmse, _ppp, _uacc, get_metric_function
+from .metrics import (_bias, _bias_slope, _conditional_bias, _crps, _crpss,
+                      _less, _mae, _mse, _msss_murphy, _nmae, _nmse, _nrmse,
+                      _pearson_r, _pearson_r_p_value, _ppp, _rmse, _std_ratio,
+                      _uacc, get_metric_function)
 from .stats import z_significance
 from .utils import check_xarray
 
+all_metrics = [
+    _pearson_r, _pearson_r_p_value, _rmse, _mse, _mae, _msss_murphy,
+    _conditional_bias, _bias, _std_ratio, _bias_slope, _crps, _crpss, _less,
+    _nmae, _nrmse, _nmse, _ppp, _uacc
+]
+
+all_metric_strings = [
```

Comment:
Same, as above `ALL_METRIC_STRINGS`

Or better yet, put them together as a `dict`
```
ALL_METRICS_DICT = {
    'pearson_r': _pearson_r
...
}
```
then down there, when you do comparisons you can do
`if metric in ALL_METRICS.keys():`